### DISCUSSION TITLE
Remove the styles applied to body and html

### DISCUSSION THREAD

**Discussion by popoleeMaster** [mantinedev/mantine#4153] [Q&A] [answered]
I am extending mantine in a custom design library.

I started by just extending the default Mantine button, apply some styles, package the library and install this package in my app.

All works fine, but when I run in my app, some style are applied to the `body` and `html` inline which cause all my existing visual to be slightly off. 

Those 2 line are added in the header, I guess by Mantine which is behind the scene using emotion.

```
 <style data-emotion="css-global" data-s></styles>
 <style data-emotion="css-global" data-s></styles>
```

as I already use tailwind in my app which reset the CSS, I do not want to have those emotion style overriding them, I am looking for a way to turn them off.

**Comment by rtivital** [comment]
You can control these styles with `withGlobalStyles` and `withNormalizeCSS` on MantineProvider

**Reply by popoleeMaster** [reply]
thanks for the reply. This remove all the globalStyles tho, is there a way to only inject a few ?

**Reply by rtivital** [reply, marked as answer]
You can apply you own global styles with theme or Global component.
